src/components/Content/index.tsx

```diff
@@ -14,7 +15,7 @@ import { PageGrid } from "components/styles/Layout";
 import { TaskStatusIconLegend } from "components/TaskStatusIconLegend";
```

Comment:
while you're here I wouldn't mind if we updated the box shadow on the icon legend to be more in line with LG's style! 🤩 

<img width="485" alt="Screenshot 2023-05-05 at 1 56 50 PM" src="https://user-images.githubusercontent.com/47064971/236532867-acd14d35-2cdb-40f9-b862-5cc8f53cb9c6.png">